tests/crypto/test_ccm_mode/testcase.yaml

```diff
@@ -0,0 +1,3 @@
+tests:
+-   test:
+        tags: crypto aes ccm
```

Comment:
I tend to agree in general with @pfalcon , but there is so much duplication in all of this with the amount of TCs that at some point this going to be out of sync and will be also confusing. There is the possibility of having a script that tries to ensure the "path" and the "tags" kinda match or calls upon the ones that don't.